### PR TITLE
Try explicit version number first

### PR DESCRIPTION
**Pull request by kofigumbs**
That didn't take long at all. Were y'all thinking of something more complex?

Iteration on #47

### CODE REVIEW

File: utils/make-pkgconfig.swift [hunk 1]
```diff
@@ -58,7 +58,7 @@ func makeFile() throws {
   let cllvmPath = pkgConfigDir.appendingPathComponent("cllvm.pc")
 
   /// Ensure we have llvm-config in the PATH
-  guard let llvmConfig = which("llvm-config") else {
+  guard let llvmConfig = which("llvm-config-3.9") ?? which("llvm-config") else {
```
Comment (CodaFi) [inline utils/make-pkgconfig.swift:61]: Other way 'round please.

**Review by CodaFi — CHANGES_REQUESTED** [review]

File: utils/make-pkgconfig.swift (hunk 1 above)
Comment (kofigumbs) [reply utils/make-pkgconfig.swift:61]: I got the opposite impression from [this comment](https://github.com/trill-lang/LLVMSwift/pull/47#issuecomment-274296002), so I'm just double checking...

Comment (CodaFi) [reply utils/make-pkgconfig.swift:61]: Oh, well.  I trust Harlan's judgement here.

**Review by CodaFi — APPROVED** [review]

**Comment by CodaFi** [comment]
Thank you kindly for the contribution (and for putting up with me through review).

⛵️

**Comment by harlanhaskins** [comment]
Actually, Keep it that way. LLVM-config-3.9 is more likely to be right

**Comment by harlanhaskins** [comment]
Thanks @hkgumbs!

Alright, to clarify.

On some systems, the default LLVM installation installs to `llvm-config-3.9` with no symlink at all. On macOS, we don't really have this problem.

Were we to use `llvm-config` first, then if it was symlinked to <3.9, it would fail after it resolved successfully, and 3.9 wouldn't be tried.

So this is correct!

**Pull request merged**